bin/describe.ml

```diff
@@ -754,7 +754,7 @@ module Preprocess = struct
     in
     let open Fiber.O in
     let+ () =
-      Process.run
+      Process.run ~display:Dune_engine.Display.quiet
```

Comment:
This should probably respect `--display` if a user runs `dune describe pp --display quiet` I would expect the setting to be respected.